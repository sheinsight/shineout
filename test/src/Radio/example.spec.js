import exampleTest from '../../example'

describe('Radio[snapshot]', () => {
  exampleTest('Radio')
})
