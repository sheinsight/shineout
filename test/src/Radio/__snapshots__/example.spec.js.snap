// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-1-base.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      <span>
        red
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        orange
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        yellow
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        green
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        cyan
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        blue
      </span>
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <span>
        violet
      </span>
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-2-group.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      green
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      cyan
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      blue
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      violet
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-3-format.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      green
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      cyan
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      blue
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      violet
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-5-block.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      green
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      cyan
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      blue
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      violet
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-6-button-1.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-6-button-2.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-6-button-3.tsx] 1`] = `
<div>
  <div>
    <label>
      <input />
      <i />
      <span>
        red
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        orange
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        yellow
      </span>
    </label>
  </div>
  <div>
    <label>
      <input />
      <i />
      <span>
        red
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        orange
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        yellow
      </span>
    </label>
  </div>
  <div>
    <label>
      <input />
      <i />
      <span>
        red
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        orange
      </span>
    </label>
    <label>
      <input />
      <i />
      <span>
        yellow
      </span>
    </label>
  </div>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-7-disabled.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      green
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      cyan
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      blue
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      violet
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-8-disabled.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      red
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      orange
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      yellow
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      green
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      cyan
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      blue
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      violet
    </span>
  </label>
</div>
`;

exports[`Radio[snapshot] Snapshot test: Radio[site/pages/components/Radio/example-9-toggle.tsx] 1`] = `
<div>
  <span>
    <label>
      <input />
      <i />
      <span>
        red
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        orange
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        yellow
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        green
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        cyan
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        blue
      </span>
    </label>
  </span>
  <span>
    <label>
      <input />
      <i />
      <span>
        violet
      </span>
    </label>
  </span>
</div>
`;
